docs/imviz/import_data.rst

```diff
@@ -109,6 +109,12 @@ Loading markers can be done with the following commands::
                                           unit="deg")]})
     viewer.add_markers(coord_i2d, use_skycoord=True, marker_name='my_markers')
 
+If you have a large catalog, you might want to filter your table to the
+marks of interest before adding them to Imviz. For instance, if your image
+has FITS WCS, you could use `astropy.wcs.WCS.footprint_contains` if you
```

Comment:
```suggestion
If you have a large catalog, you might want to filter your table to the
marks of interest before adding them to Imviz, in order to avoid performance
issues associated with adding large numbers of markers. For instance, if your 
image has FITS WCS, you could use `astropy.wcs.WCS.footprint_contains` if you
```

I think it's worth making clear that you would do this to avoid potential performance issues/errors like Harry ran into.